docs/01-engage/02-accounts/02-creating-accounts/index.md

```diff
@@ -0,0 +1,113 @@
+
+:::caution
+For incorrect and incomplete phone numbers, a **validation pop-up error message** will appear!
+:::
+
+![Create Account Phone Number](./create-account-3.png)
+
+4. In the *Email Address* section, input your *Personal* or *Work* email and select it respectively. You can also add extra email addresses with **+ Add Email Address** and remove them with the **Remove** button.
+
+![Create Account Email](./create-account-4.png)
+
+5. After this, in the *Online Presence* section, select the type from the drop-down including *Facebook*, *LinkedIn* and *Instagram* etc and state its example. You can add additional types using **Add Profile** and remove them as wanted with the **Remove** option.
+
+:::note 
+Online Presence refers to the combined existence of an individual or organization that is found online via search. 
+:::
+
+![Create Account Online Presence](./create-account-5.png)
+
+6. For the *Contact Preferences* section, select the way you want to receive communication i.e. via **Email**, **Post**, **SMS** or **Telephone**. Each category contains five different options named **Unspecified**, **Admin Only**, **Fundraising Only**, **All** and **None**. Select any category. For details, refer to the Contact Preferences(link to page) page.
+
+:::note Important
+You can select two categories at one time by using the **Opt-In** and **Opt-Out** options.
+:::
+
+![Create Account Contact Preferences](./create-account-6.png)
+
+7. Next, if you want to add a *Gift*, click **Eligibility** as *Yes*. Then add the **Declaration Date**, **Valid from**, choose **Declaration Source** and attach the source files if needed. For details, refer to Gift Aid Section(link).
+
+![Create Account Gift Aid](./create-account-7.png)
+
+8. In the *interactions* section, assign any **Touchpoint** for analytics purposes. Select the **Date** and **Time**. For more information, go to Interactions Section(link).  
+
+![Create Account Interactions](./create-account-8.png)
+
+9.  Click **+ Create Account**.
+
+## Create an Organizational Account
+
+:::note Important
+**Gift Aid** is not eligible for an organizational account.
+:::
+
+To create an individual account, click **organization** on the toggle in the right-hand side.
+
+![Toggle Button](./toggle-button.png)
+
+1. In the *Name* section, input the **Organization Name** and select the **Organization Type** from the drop-down.
+
+![Organization Name](./organization-name.png)
+
+2. Choose the **Title**, **First Name** and **Last Name** of the respective *Contact Person*. 
+
+![Contact Person](./contact-person.png)
+
+3. Rest of the sections including *Address, Phone Number, Email Address, Online Presence, Contact Preferences and Interactions* are added the same way as for an individual account. You can follow the steps mentioned above.
+
+## You Did It!
+
+Your account is now setup in ENGAGE. Get started and donate as much as you want!
```

Comment:
Please take this out. It is too informal and also the account doesn't belong to the person entering it so it is not like a website self-service instructions.